packages/expo-sensors/src/__tests__/Gyroscope-test.ios.ts

```diff
@@ -1,12 +1,11 @@
-import ExponentGyroscope from '../ExponentGyroscope';
 import Gyroscope from '../Gyroscope';
 
 afterEach(() => {
   Gyroscope.removeAllListeners();
 });
 
 it(`adds an "gyroscopeDidUpdate" listener`, () => {
-  const NativeGyroscope = ExponentGyroscope;
+  const NativeGyroscope = Gyroscope._nativeModule;
```

Comment:
What was the problem with `ExponentGyroscope` (and others)?

Reply:
There was no problem, I just think that it's better in those test cases to refer to the native module attached to the main sensor rather than separately imported.